packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -7040,7 +7040,7 @@ export default class Meeting extends StatelessWebexPlugin {
       shareAudioEnabled = true,
       shareVideoEnabled = true,
       remoteMediaManagerConfig,
-      bundlePolicy,
+      bundlePolicy = 'max-bundle',
```

Comment:
Should we change it also for the WCME? I believe WCME is still max-compat by default

Reply:
yes, we should change it in WCME too. then I wouldn't even have to change it in SDK, but it's just quicker to do 1 PR in SDK then 3 in WCME, internal-media-core and SDK